docs/open-telemetry.asciidoc

```diff
@@ -0,0 +1,81 @@
+[[opentelemetry]]
+=== Using OpenTelemetry
+
+You can use https://opentelemetry.io/[OpenTelemetry] to monitor the performance and behavior of your Elasticsearch requests through the Ruby Client.
```

Comment:
Not super-important but there is a [shared attribute](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc) for `Elasticsearch`. Please take or leave it as you see fit.
```suggestion
You can use https://opentelemetry.io/[OpenTelemetry] to monitor the performance and behavior of your {es} requests through the Ruby Client.
```